firestore.rules

```diff
@@ -0,0 +1,9 @@
+rules_version = '2';
+service cloud.firestore {
+  match /databases/{database}/documents {
+    match /{document=**} {
+      allow read, write: if
+          request.time < timestamp.date(2021, 4, 4);
```

Comment:
Why is there this time rule here? I'm not sure I understand why we need to be less than 4.4.2021

Reply:
It's a generated automatically by Firebase, it allows everything for a month.
We'll have to change it and specify real permissions when we'll have user authentication.